src/bin/cli.rs

```diff
@@ -0,0 +1,39 @@
+use structopt::StructOpt;
+
+use trad::handlers::echo_handler::EchoHandler;
+
+#[derive(Debug, StructOpt)]
+enum Command {
+    Echo {
+        #[structopt(short = "m", long = "message")]
+        message: String,
+    },
+}
+
+#[derive(Debug, StructOpt)]
+#[structopt(
+    name = "Trad-terminal",
+    about = "A faster way to interact with TRAD App"
+)]
+struct CliOpt {
+    #[structopt(subcommand)]
+    command: Command,
+
+    // -d or --debug, true if it is present
+    #[structopt(short = "d", long = "debug")]
+    debug: bool,
+}
+
+fn main() {
```

Comment:
Can we add this part with args as part of main.rs?
In that case we can run binary without args in interactive mode and with args for quick note.

Reply:
Done!